mapreduce/src/test/java/org/pentaho/hadoop/mapreduce/PentahoMapReduceIT.java

```diff
@@ -92,8 +96,8 @@ public void testMapRunnable_WordCount() throws IOException, KettleException, URI
     List<String> wordsToCalculate =
       IntStream.rangeClosed( 1, ROWS_TO_CALCULATE ).mapToObj( value -> String.valueOf( WORDS_TO_CALCULATE ) )
         .collect( Collectors.toList() );
-    if ( DEBUG_MODE ) {
-      System.out.println( "Mapper input data: " + ROWS_TO_CALCULATE + " rows of [" + WORDS_TO_CALCULATE + "]" );
+    if ( log.isBasic() ) {
```

Comment:
This should still be log.isDebug, not basic.  I believe the original point of the code was to only log if the DEBUG_MODE flag was changed in the code and to be silent otherwise to avoid spamming the build server logs.

Reply:
Please see the latest version it was changed in different way as it was DEBUG_MODE = false; so it was  always false.
So I changed to set log level to basic and changed if to:
if ( log.isBasic() ) {
      log.logBasic( ...
    }

Reply:
Changed:
if ( log.isBasic() ) {
log.logBasic( ...
}
to
if ( log.isDebug() ) {
log.logDebug( ...
}